### PR TITLE
crypto: add example ECDSA contract produces event

### PR DESCRIPTION
**Pull request by joyqvq**
end to end testing: https://gist.github.com/joyqvq/89aa06ee3b97b66323568770ef9d4dfb

### CODE REVIEW

**Review by randall-Mysten — APPROVED** [review]

**Comment by Clay-Mysten** [comment]
Thanks so much for the new example, @joyqvq ! Should I list it here?
https://docs.sui.io/explore/examples

**Comment by Clay-Mysten** [comment]
> Thanks so much for the new example, @joyqvq ! Should I list it here? https://docs.sui.io/explore/examples

I took the initiative:
https://github.com/MystenLabs/sui/pull/4315

I didn't want to forget. Reviewing the draft you sent me now. Thanks, @joyqvq !

**Comment by joyqvq** [comment]
> Thanks so much for the new example, @joyqvq ! Should I list it here? https://docs.sui.io/explore/examples

yes - im still making some edits to the example contract, will ping you when this is finalized!

File: sui_programmability/examples/math/sources/ecdsa.move [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// Basic ECDSA util contract, emits an event with data with the output of the crypto function
```
Comment (kchalkias) [inline sui_programmability/examples/math/sources/ecdsa.move:4]: Prefer adding fullstops (periods) in every comment.

**Review by kchalkias — APPROVED** [review]

**Pull request merged**